fc3/settings/base.py

```diff
@@ -78,7 +78,6 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.doc.XViewMiddleware',
```

Comment:
I did see "django.middleware.doc.XViewMiddleware is removed." in https://docs.djangoproject.com/en/1.8/releases/1.8/.

However, further research indicates XViewMiddleware is required for Django's admindocs. We support admindocs in cracklyfinger to assist in development. Apparently the code was moved. https://docs.djangoproject.com/en/1.8/releases/1.6/ says

> The XViewMiddleware has been moved from django.middleware.doc to django.contrib.admindocs.middleware because it is an implementation detail of admindocs, proven not to be reusable in general."